src/utils/stringUtils.ts

```diff
@@ -75,7 +75,7 @@ export class stringUtils {
      * @param string string that you want to checks that's url or not
      * @returns bool returns if it were url or not
      */
-    public static StrisURL(string: string) {
+    public static strIsURL(string: string) {
```

Comment:
Rename just to isURL, and once more and again... parameter name has to be url...